scripts/scil_compute_sf_from_sh.py

```diff
@@ -43,14 +44,15 @@ def _build_arg_parser():
     p.add_argument("--extract_as_dwi", action="store_true",
                    help="Generate a DWI-like output, including a `.bval` file "
                         "and b0 images in the sf file.")
-    p.add_argument('--bval', required=True,
+    p.add_argument('--bval',
```

Comment:
I really don't undestand why bvecs is automatically saved (with a default filename we can't change), but bval is optional and we can choose the name.